.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# Set update schedule for GitHub Actions
+# See https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/keeping-your-actions-up-to-date-with-dependabot
+
+version: 2
+updates:
+
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    # Check for updates to GitHub Actions every weekday
+    interval: "daily"
```

Comment:
monthly? I find all of the updates a bit crazy...

Reply:
This one came from one of the other repos, I'm not actually sure if dependabot works on mono repos...

Reply:
It does for actions – but you'll be limited by what mono_repo has setup

Reply:
hmm, maybe should just drop it then.

Reply:
so far I've only seen this as noise – especially for monorepos that use mono_repo